Python/Product/PythonTools/PythonTools/Navigation/EditFilter.cs

```diff
@@ -658,7 +658,10 @@ class NodeEnumerator<T> : IVsEnumNavInfoNodes where T : IVsNavInfoNode {
                             var line = pyPoint.Value.GetContainingLine();
                             var lineText = line.GetText();
```

Comment:
This is the code that adds the comment marker on the next line when you break a comment on the current line.

With this change, it can only be embedded in a multiline string literal and it's at the start of the line, so I'm inclined to keep the behaviour for now and see whether people notice. But I'll add the comment here

---
In reply to: [75517787](https://github.com/Microsoft/PTVS/pull/1536#discussion_r75517787) [](ancestors = 75517787)